src/datachain/remote/studio.py

```diff
@@ -231,6 +235,38 @@ def _unpacker_hook(code, data):
 
         return msgpack.ExtType(code, data)
 
+    async def tail_job_logs(self, job_id: str) -> AsyncIterator[dict]:
+        """
+        Follow job logs via websocket connection.
+
+        Args:
+            job_id: ID of the job to follow logs for
+
+        Yields:
+            Dict containing either job status updates or log messages
+        """
+        parsed_url = urlparse(self.url)
+        ws_url = urlunparse(parsed_url._replace(scheme="ws"))
```

Comment:
Let's not use WebSockets without TLS, please. At least, let's match the scheme from `self.url` converting `http` to `ws` and `https` to `wss` to simplify local development.

```suggestion
        ws_url = urlunparse(parsed_url._replace(scheme="wss"))
```